StardewArchipelago/ModEntry.cs

```diff
@@ -66,8 +66,9 @@ public class ModEntry : Mod
         private NightShippingBehaviors _shippingBehaviors;
 
         private ModRandomizedLogicPatcher _modLogicPatcher;
-        private CallableModData _callableModData;
+        private InitialModGameStateInitializer _callableModData;

```

Comment:
If you believe the new name is better for the class, the instance name should probably also change